src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -90,6 +89,9 @@ public void setRole(final AbstractRole role) {
 
     @Override
     public void update() {
+        // Registers every time because it might be registered to an old instance of the event bus
+        // (caused by saving/loading).
+        Pawntastic.BUS.register(this);
```

Comment:
Question
I'm guessing we couldn't solve this in a better way? Is there no way to force a method to be called after it has been loaded?

Reply:
Actually took a second look at it because thought we really cant be the only ones that want this. And found an 11year old stackoverflow thread about it this time. So should be better now 👍

Reply:
The `@Serial
    private void readObject` solution is really elegant, nicely done!